angular/core/models/notification_model.coffee

```diff
@@ -16,5 +16,9 @@ angular.module('loomioApp').factory 'NotificationModel', (BaseModel) ->
     group: ->
       @event().group()
 
+    actionPath: ->
+      switch @kind()
+        when 'motion_closed', 'motion_closed_by_user' then 'outcome'
```

Comment:
As these get more complicated, we'll want to make a `actionPathMap` in `notification_model.coffee`, so this becomes

```
actionPath: ->
  @constructor.actionPathMap[@kind()]
```

which will keep this a bit cleaner